doc/python/static-image-export.md

```diff
@@ -41,31 +41,20 @@ Plotly figures are interactive when viewed in a web browser: you can hover over
 
 
 <!-- #region -->
-#### Install Dependencies
-Static image generation requires the [orca](https://github.com/plotly/orca) commandline utility and the [psutil](https://github.com/giampaolo/psutil) and [requests](https://2.python-requests.org/en/master/) Python libraries. There are 3 general approach to installing these dependencies.
-
-##### conda
-Using the [conda](https://conda.io/docs/) package manager, you can install these dependencies in a single command:
+#### Install Dependencies (Kaleido)
+Static image generation requires the [`kaleido`](https://github.com/plotly/Kaleido) package which can be installed using pip...
```

Comment:
strictly speaking this isn't correct, it requires "either Kaleido (recommended) or Orca (deprecated)" or something like that, no? Just to avoid confusing people who already know Orca? (same comment applies in other places in this PR where we've said Kaleido is required)

Reply:
I can't select it but on line 40 JEPG should be JPEG.

Reply:
Thanks:
 - JPEG change (a033df0c3e10f550d7d8df0448e43bacd1eb49e6)
 - "Write Image File" changes (2bdcfe210865695b9db89e64c5f55e6c2c039be8)
 - Merged install dependency sections and better explain that orca is legacy and kaleido is new (d1fc9a8a98830abdef33bbb15ef2b9ee56228023)